eng/tools/publish-to-npm.ps1

```diff
@@ -123,16 +123,18 @@ try {
         npm config set $regAuth`:email=not_set
     }
 
+    $publishList = @()
     foreach ($p in $packageList) {
-        if($p.Publish) {
+        if($p.Publish -and !$publishToNpm) {
+            # Publishing to private feed
             if ($tag) {
-              Write-Host "npm publish $($p.TarGz) --access=$accessLevel --registry=$registry --always-auth=true --tag=$tag"
-              npm publish $p.TarGz --access=$accessLevel --registry=$registry --always-auth=true --tag=$tag
+                Write-Host "npm publish $($p.TarGz) --access=$accessLevel --registry=$registry --always-auth=true --tag=$tag"
+                npm publish $p.TarGz --access=$accessLevel --registry=$registry --always-auth=true --tag=$tag
             }
             else {
-              Write-Host "Tag is empty"
-              Write-Host "npm publish $($p.TarGz) --access=$accessLevel --registry=$registry --always-auth=true"
-              npm publish $p.TarGz --access=$accessLevel --registry=$registry --always-auth=true
+                Write-Host "Tag is empty"
+                Write-Host "npm publish $($p.TarGz) --access=$accessLevel --registry=$registry --always-auth=true"
+                npm publish $p.TarGz --access=$accessLevel --registry=$registry --always-auth=true
```

Comment:
Are we handling exit codes here? I think we should `$PSNativeCommandUseErrorActionPreference`?

Reply:
error action = stop, psnative = true